src/containers/pages/InterventionPlan/PlanDefinitionList/tests/index.test.tsx

```diff
@@ -291,32 +305,34 @@ describe('components/InterventionPlan/PlanDefinitionList', () => {
         list: mockList,
       };
     });
-    store.dispatch(fetchPlanDefinitions(fixtures.plans));
+
+    store.dispatch(fetchPlanRecords(extractedPlanRecords));
     const userName = 'macTavish';
     store.dispatch(fetchPlansByUser([fixtures.plans[0]], userName));
     const props = {
-      fetchPlans: jest.fn(),
       history,
       location: {
+        hash: '',
         pathname: PLAN_LIST_URL,
         search: `?user=${userName}`,
+        state: undefined,
       },
       match: {
         isExact: true,
         params: {},
         path: `${PLAN_LIST_URL}`,
         url: `${PLAN_LIST_URL}`,
       },
-      service: mockClass,
+      serviceClass: mockClass,
     };
     const wrapper = mount(
       <Provider store={store}>
         <Router history={history}>
-          <ConnectedPlanDefinitionList {...props} />
+          <PlanDefinitionList {...props} />
         </Router>
       </Provider>
     );
-    await new Promise(resolve => setImmediate(resolve));
+    await flushPromises();
```

Comment:
lets reaudit the tests and wrap wrapper.update  calls due to async calls inside act.

Reply:
have wrapped tests updating component status with act